src/Adapter/Module/QueryHandler/GetModuleInfosHandler.php

```diff
@@ -50,6 +51,10 @@ public function handle(GetModuleInfos $query): ModuleInfos
             throw new ModuleNotFoundException('Cannot find Module with name ' . $query->getTechnicalName()->getValue());
         }
 
+        if (!$module->database->get('installed')) {
```

Comment:
There are cases where we'll want to block some actions for an uninstalled module, but not this particular case Because it is interesting to know that the module is present, not installed and get its current version for example, so this check should be removed here